liquibase-extension-examples/src/main/resources/META-INF/services/liquibase.executor.Executor

```diff
@@ -0,0 +1 @@
+liquibase.executor.jvm.ExampleExecutor
```

Comment:
Should this class be committed?

Reply:
It should be okay.  The liquibase-extension-examples are not shipped.